src/__internal__/focus-trap/focus-trap.component.js

```diff
@@ -13,19 +14,21 @@ import {
   isRadio,
   setElementFocus,
 } from "./focus-trap-utils";
+import { ModalContext } from "../../components/modal/modal.component";
 
 const FocusTrap = ({
   children,
   autoFocus = true,
   focusFirstElement,
   bespokeTrap,
   wrapperRef,
+  isAnimationComplete = false,
```

Comment:
Is this used?

Reply:
How can it be legacy when you've added it here?